documentation/docs/examples/cad-colors.mdx

```diff
@@ -51,6 +51,6 @@ subtree. Reveal supports this through the use of `Cognite3DModel.setNodesColorsB
 set to `true`:
 <LiveCodeSnippet>
 {`
-model.setNodeColorByTreeIndex(122, 240, 100, 60, true /* apply to children */);
+model.setNodeColorByTreeIndex(122, 240, 100, 60, applyToChildren=true);
```

Comment:
Unrelated change which I accidentally added to the commit, but I want to change this so I'll leave it.